airflow/serialization/serialized_objects.py

```diff
@@ -840,7 +840,7 @@ def deserialize(cls, encoded_var: Any, use_pydantic_models=False) -> Any:
             args = deser["args"]
             kwargs = deser["kwargs"]
             del deser
-            exc_cls = import_string(f"airflow.exceptions.{exc_cls_name}")
+            exc_cls = import_string(exc_cls_name)
```

Comment:
Note: Needed to remove prefix and expose the full class name incl. module in order allowing to also support all inherited classes from AirflowException also in custom modules/provider packages.
All exceptions seen in the codebase were adjusted with this commit.